doozer

```diff
@@ -1590,6 +1675,40 @@ installonly_limit=3
         green_print('All repos synced to {}'.format(output))
 
 
+@cli.command("config:update-required", short_help="Update images that are required")
+@click.option("--image-list", help="File with list of images, one per line.", required=True)
+@pass_runtime
+def config_update_required(runtime, image_list):
+    """Ingest list of images and update data repo
+    with which images are required and which are not.
+    """
+    _fix_runtime_mode(runtime)
+    runtime.initialize(**CONFIG_RUNTIME_OPTS)
+
+    with open(image_list, 'r') as il:
+        image_list = il.readlines()
+        image_list = [i.strip() for i in image_list if i.strip()]
```

Comment:
why not
```suggestion
        image_list = [i.strip() for i in il.readlines() if i.strip()]
```
?

Reply:
Old habits die hard...to me your version is less readable. Doing everything inline is "clever" which, yes, is more terse but a little harder to follow. But I'm not gonna fall on my sword here, so I changed it anyways.

Reply:
No need to die on this hill LOL
For me, in this case, it's not *just* brevity... certainly if it was more complicated than `il.readlines()` it would make sense to hold it in an intermediate variable. But here you had the same variable for two semantically different uses... lines coming out of a file versus image names. Inlining keeps me from the interrupt of looking up a line to see what's in there. Yes, my attention span is that short :laughing: So thanks!